pkgs/tools/text/repgrep/default.nix

```diff
@@ -32,8 +32,13 @@ rustPlatform.buildRustPackage rec {
 
     pushd "$(dirname "$(find -path '**/repgrep-stamp' | head -n 1)")"
     installManPage rgr.1
-    installShellCompletion rgr.{bash,fish} _rgr
     popd
+
+    # As it can be seen here: https://github.com/acheronfail/repgrep/blob/0.15.0/.github/workflows/release.yml#L206, the completions are just the same as ripgrep
```

Comment:
Could we instead use the ripgrep files to avoid the cross compilation issue?

Reply:
`ripgrep` also uses [`installShellCompletion --cmd`](https://github.com/NixOS/nixpkgs/blob/73365029fd28c80c84f5425d0d6f6387c1faff08/pkgs/tools/text/ripgrep/default.nix#L37). The options I see to avoid cross-compilation issues are:

1. Adding `buildPackages` as a pattern and run `ripgrep` with `${stdenv.hostPlatform.emulator buildPackages}`; or
2. Make the shell completion optional with `lib.strings.optionalString`, giving `(stdenv.hostPlatform.canExecute stdenv.buildPlatform)` as its condition.

Reply:
I like the second one.